io.github.tobagin.scramble.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.github.tobagin.scramble
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: scramble
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-documents:rw
+  - --filesystem=xdg-desktop:rw
+  - --filesystem=xdg-download:rw
+
+modules:
+  - name: blueprint-compiler
+    buildsystem: meson
+    cleanup:
+      - "*"
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/blueprint-compiler.git
+        tag: v0.18.0
+        commit: 07c9c9df9cd1b6b4454ecba21ee58211e9144a4b
+
+  - name: inih
+    buildsystem: meson
+    config-opts:
+      - -Ddistro_install=true
+      - -Dwith_INIReader=true
+    sources:
+      - type: git
+        url: https://github.com/benhoyt/inih.git
+        tag: r62
+        commit: 26254ee9de7681f8825433415443e7116ff24b98
+
+  - name: exiv2
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DEXIV2_ENABLE_NLS=ON
+      - -DEXIV2_BUILD_SAMPLES=OFF
+      - -DEXIV2_BUILD_EXIV2_COMMAND=OFF
+      - -DEXIV2_ENABLE_BMFF=OFF
```

Comment:
so you don't want to handle HEIF?

Reply:
there is no reason to set this to OFF. It's an option that shouldn't exist upstream.